### PR TITLE
Fixed inconsistencies: TS issues, docs and a URL cleaner

### PR DESCRIPTION
**Pull request by robinpiets**
This PR fixes some TS issues, docs and a new function to `removeQueryStrings` from an URL.

### CODE REVIEW

File: packages/client/src/components/Layouts/DefaultLayout.test.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Wrapper = ({
   formTitle,
   heading,
 }: {
-  children: React.ReactNode;
+  children: React.ReactChildren;
```
Comment (afjlambert) [inline packages/client/src/components/Layouts/DefaultLayout.test.tsx]: Can you do?

```
type Props = {
  formTitle: string;
  heading: string;
}

const Wrapper: React.FC<Props> = ({
```

Comment (afjlambert) [reply packages/client/src/components/Layouts/DefaultLayout.test.tsx]: So without the `children`

**Review by afjlambert — CHANGES_REQUESTED** [review]
Small tweak.

File: packages/client/src/components/Layouts/DefaultLayout.test.tsx (hunk 1 above)
Comment (robinpiets) [reply packages/client/src/components/Layouts/DefaultLayout.test.tsx]: Fixed (-:

**Review by svenjens — APPROVED** [review]

**Pull request merged**